src/main/resources/application.properties

```diff
@@ -5,6 +5,9 @@ spring.datasource.username=${DB_USER}
 spring.datasource.password=${DB_PASSWORD}
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 server.port=8080
-server.servlet.context-path=/api
 spring.jpa.hibernate.ddl-auto=validate
+server.servlet.context-path=/api
 spring.jpa.show-sql=true
+
+jwt.expiration=300000
+jwt.secret=wefwqfdfwqfq4f3fq3af43453452ef34fewfewfe
```

Comment:
It's better to get sensitive properties from system properties and not pushing it to git.